pages/success-banner/index.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import Head from 'next/head';
 
-class Button extends React.Component {
+const eventLog = (eventDesc) => {
+  const data = 'Banner ' + eventDesc;
+
+  return () => {
+    const url = new URL(location.href);
+
+    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
```

Comment:
Hmmm, what's this for? Isn't Etcher 1.0.0 the only version that contains code to display this banner anyway? Or is this related to the Robot-changes in https://github.com/resin-io/etcher/pull/1422 which were performed after the release of Etcher 1.0.0?